client/signup/steps/user/style.scss

```diff
@@ -0,0 +1,24 @@
+body.is-section-signup.is-white-signup .signup.is-onboarding {
+	$gray-40: #787c82;
```

Comment:
We can also use --studio-gray-40 here since it is already defined.

Reply:
@aneeshd16 Good catch! Updated!